### PR TITLE
Add CORS headers

### LINKED ISSUE #74: Add CORS headers
As the service will probably almost never run on the same domain and port as the client application we need to set CORS headers.

From what I've seen, this should not be too much work:
https://stackoverflow.com/a/38162454

I guess the trouble starts with the configuration. The - IMO - ideal solution would be to have the configuration in the Docker Compose file. But not sure, if that makes sense :smile:

### PR DESCRIPTION
**Pull request by open-dynaMIX**
Configuration is possible with the env vars:
 * CORS_ORIGIN_ALLOW_ALL
 * CORS_ORIGIN_REGEX_WHITELIST

Closes #74

### CODE REVIEW

File: document_merge_service/settings.py [hunk 1]
```diff
@@ -97,6 +99,17 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
 }
 
 
+# CORS
+CORS_ORIGIN_ALLOW_ALL = env.bool("CORS_ORIGIN_ALLOW_ALL", False)
+CORS_ORIGIN_REGEX_WHITELIST = (
+    (
+        r"{0}".format(
+            env.str("CORS_ORIGIN_REGEX_WHITELIST", r"^(http://)?127\.0\.0\.1:\d{4}$")
```
Comment (winged) [inline document_merge_service/settings.py]: Shouldn't we make this an `env.list`, so multiple values can be configured?

The localhost ones could probably be hardcoded above

File: README.md [hunk 2]
```diff
@@ -93,6 +93,10 @@ Per default no authentication is needed. To protect api, integrate it with your
 * `CACHE_BACKEND`: [cache backend](https://docs.djangoproject.com/en/1.11/ref/settings/#backend) to use (default: django.core.cache.backends.locmem.LocMemCache)
 * `CACHE_LOCATION`: [location](https://docs.djangoproject.com/en/1.11/ref/settings/#std:setting-CACHES-LOCATION) of cache to use
 
+#### CORS
+* `CORS_ORIGIN_ALLOW_ALL`: [allow all](https://github.com/ottoyiu/django-cors-headers#cors_origin_allow_all)
+* `CORS_ORIGIN_REGEX_WHITELIST`: [regex whitelist](https://github.com/ottoyiu/django-cors-headers#cors_origin_regex_whitelist) defaults to "^(http://)?127\.0\.0\.1:\d{4}$"
+
```
Comment (winged) [inline README.md]: Please add a note that users with nginx/apache in front should ensure the CORS configurations match (or are deferred to the service on principle)

File: document_merge_service/settings.py (hunk 1 above)
Comment (open-dynaMIX) [reply document_merge_service/settings.py]: Done

File: README.md (hunk 2 above)
Comment (open-dynaMIX) [reply README.md]: Done

**Pull request merged**